### PR TITLE
Lets config to parse config

### PR DESCRIPTION
**Pull request by przepompownia**

### CODE REVIEW

**Comment by przepompownia** [comment]
It bases on #17 
Parsing the custom configuration seems to be a responsibility of `config`.
This PR allow to avoid parsing it again. I found one place that reveal such potential situation and would like to prepare next PR in free time.

File: src/server/scan.ts [hunk 1]
```diff
@@ -119,18 +119,6 @@ process.on('message', (mess) => {
...
-    }
-    if (config.count !== undefined) {
-      count = config.count
-    }
-    if (config.workDirPatterns !== undefined
-        && Array.isArray(config.workDirPatterns)
-        && config.workDirPatterns.length
-       ) {
-      customWorkDirPatterns = config.workDirPatterns
-    }
```
Comment (iamcco) [inline src/server/scan.ts]: why these lines are deleted?

Comment (przepompownia) [reply src/server/scan.ts]: Please compare this branch against the current master. This logic was moved to `config`.

At this moment, github keeps outdated state of master and shows a conflict so please check it locally.

**Comment by przepompownia** [comment]
Why config's getter does not work when it is called from subprocess? It's prpbably some basic question, specific to a beginner ;)

**Comment by przepompownia** [comment]
It's my mistake. I deleted local assignments from config. The last commit restores these assignments without conditions on `projectRootPatterns` that are redundant in this place.

**Pull request merged**